ensime_shared/ensime.py

```diff
@@ -279,11 +336,15 @@ def au_buf_leave(self, client, filename):
 
     @execute_with_client()
     def au_cursor_hold(self, client, filename):
-        client.on_cursor_hold(filename)
+        """Handler for CursorHold autocommand event."""
+        self.watchdog_client_server_connection(client)
+        client.unqueue_and_display(filename)
+        client.editor.cursorhold()
```

Comment:
Okay, my issue is that I have a deoplete delay of 450ms that was fired all the time. I removed it. I'll file an issue to keep track of the "blocked UI" issue, as I've experienced it in other ensime requests that take a lot of time to come back.